csaf-validation/src/main/kotlin/io/github/csaf/sbom/validation/requirements/Requirements.kt

```diff
@@ -83,27 +131,35 @@ object Requirement7 : Requirement {
 }
 
 object Requirement8 : Requirement {
-    override fun check(ctx: ValidationContext): ValidationResult {
-        // TODO: actually implement the requirement
-        return ValidationSuccessful
-    }
+    override fun check(ctx: ValidationContext) =
+        if (ctx.dataSource == ValidationContext.DataSource.SECURITY_TXT) {
+            ValidationSuccessful
```

Comment:
I don't think that this requirement is checked properly here. Please add a TODO at least. I'd appreciate resolving this in a separate PR. Also, the link to the requirement spec would be great.

Reply:
I think it is. What else should we check here? The rest of the behavior is already done in the fetching part. Added the spec link.